psi4/src/psi4/libscf_solver/rhf.cc

```diff
@@ -87,9 +87,10 @@ void RHF::common_init()
     // Allocate matrix memory
     Fa_        = SharedMatrix(factory_->create_matrix("F"));
     Fb_        = Fa_;
-    Ca_        = SharedMatrix(factory_->create_matrix("C"));
+    Ca_        = SharedMatrix(factory_->create_matrix("MO coefficients (C)"));
```

Comment:
`fchk` will work even if the names were different. For example, If one looks at mp2.cc inside dfmp2, ```Ca_ = SharedMatrix(new Matrix("DF-MP2 Natural Orbitals", nsopi_, nmopi_));```
I wanted this to be reflected when we write the wfn's info to the `fchk_writer` If the wfn doesn't do anything special, the names would stay the same as reference wfn's. Does this make sense?